packages/api/lambdas/message-consumer.js

```diff
@@ -166,10 +170,13 @@ function processRecord(record, fromSNS) {
  * @returns {(error|string)} Success message or error
  */
 function handler(event, context, cb) {
-  const fromSns = event.Records[0].EventSource === 'aws:sns';
   const records = event.Records;
 
-  return Promise.all(records.map((r) => processRecord(r, fromSns)))
+  // fetch enabled rules from the API and cache in memory so we don't need a ton of DB connections
+  return fetchEnabledRules()
+    .then((rules) => Promise.all(records.map(
+      (r) => processRecord(r, (r.EventSource === 'aws:sns'), rules)
```

Comment:
```suggestion
      (record) => processRecord(record, (record.EventSource === 'aws:sns'), rules)
```

🐜 Nit pick about one letter variable names